docs/pages/distribution/publishing-websites.mdx

```diff
@@ -176,68 +176,56 @@ Install the [Vercel CLI](https://vercel.com/docs/cli).
   Configure redirects for single-page applications.
   <Tabs>
     <Tab label="Expo Router">
-      > If your app uses [static rendering](/router/reference/static-rendering), then you can skip this step.
 
-      `expo.web.output: 'single'` generates a single-page application. It means there's only one **dist/index.html** file to which all requests must be redirected. This can be done in Vercel by creating a **./public/vercel.json** file and redirecting all requests to **/index.html**.
+      Create a `vercel.json` file in the root of your app and add the following:
 
-      ```json public/vercel.json
+      ```json vercel.json
       {
-        "version": 2,
+        "buildCommand": "expo export -p web",
+        "outputDirectory": "dist",
+        "devCommand": "expo",
+        "cleanUrls": true,
+        "framework": null,
         "rewrites": [
           {
-            "source": "/(.*)",
+            "source": "/:path*",
             "destination": "/"
           }
         ]
       }
       ```
-      If you modify this file, you will need to rebuild your project with `npx expo export -p web` to have it safely copied into the **dist** directory.
+
+      If your app uses [static rendering](/router/reference/static-rendering), then you may want to add additional [dynamic route configuration](/router/reference/static-rendering#dynamic-routes).
+
     </Tab>
     <Tab label="webpack">
-      If your app implements any navigation, you must configure Vercel to redirect requests to the single **web-build/index.html** file. This can be done in Vercel by creating a **`./public/vercel.json` file and redirecting all requests to `/index.html`.**
-
-      Navigate inside the **web-build** directory and run the following command to create **\_redirects** file with rule:
+      Create a `vercel.json` file in the root of your app and add the following:
```

Comment:
```suggestion
      Create a **vercel.json** file at the root of your app and add the following:
```

- Small suggestion: [Filenames as bold text](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)
- Also, another suggestion to replace in with "at" to specify a specific location (which is the root of project's directory)